libmamba/data/mamba.nu

```diff
@@ -0,0 +1,23 @@
+
+export def micromamba [arg] {
+  let cmd = arg[0]
+  let argv = arg[1]
+
+  if ($cmd in [ activate deactivate ]){
+        $env.MAMBA_EXE $cmd $argv
```

Comment:
I don't understand how this works. For the other shells we use `source` here because we want to change the current shell environment

Reply:
Ah I believe you are right. I took this from the fish shell code. I am not too familiar with fish but I believe there it is handled a bit differently. I do some more testing this weekend.

Reply:
I had a closer look at your earlier remark. The file above makes the micromamba function available to the path. There are some changes to nushell that now deprecates the `let-env` command. I have fixed this with the latest commit.